gen/dcos-config-windows.yaml

```diff
@@ -255,3 +255,110 @@ package:
   - path: {{ windows_dcos_install_path }}\etc\adminrouter\adminrouter-listen-open.conf
     content: |
         listen 61001 default_server;
+
+  - path: {{ windows_dcos_install_path }}\etc\mesos\modules\logsink.json
+    content: |
+      {
+        "libraries": [
+          {
+            "file": "logsink.dll",
+            "modules": [
+              {
+                "name": "com_mesosphere_mesos_LogSink",
+                "parameters": [
+                  {
+                    "key": "output_file",
+                    "value": "nul"
```

Comment:
I'm not quite getting: where does the custom glog logsink (implemented in this module) dump logs on a Windows agent host?

Reply:
@DominikDary I guess we need to find out someone who can address that question.

Reply:
this has been discussed in slack with @asekretenko, @kaysoky and  @akornatskyy and the logsink module should be removed from windows and as well this config file.
This with https://github.com/dcos/dcos/pull/6917/files#r394897271 is the last piece that needs to be done in order to merge this Pr.